doc/03_reference/cs_registers.rst

```diff
@@ -140,9 +140,77 @@ Machine ISA Register (misa)
++-------------+------------+------------------------------------------------------------------------+
+| 17          | RO   (0x0) | **R** (Reserved)                                                       |
++-------------+------------+------------------------------------------------------------------------+
+| 16          | RO   (0x0) | **Q** (Quad-precision floating-point extension)                        |
++-------------+------------+------------------------------------------------------------------------+
+| 15          | RO   (0x0) | **P** (Tentatively reserved for Packed-SIMD extension)                 |
++-------------+------------+------------------------------------------------------------------------+
+| 14          | RO   (0x0) | **O** (Reserved)                                                       |
++-------------+------------+------------------------------------------------------------------------+
+| 13          | RO   (0x0) | **N** (User-level interrupts supported)                                |
++-------------+------------+------------------------------------------------------------------------+
+| 12          | RO   (0x1) | **M** (Integer Multiply/Divide extension)                              |
++-------------+------------+------------------------------------------------------------------------+
+| 11          | RO   (0x0) | **L** (Tentatively reserved for Decimal Floating-Point extension)      |
++-------------+------------+------------------------------------------------------------------------+
+| 10          | RO   (0x0) | **K** (Reserved)                                                       |
++-------------+------------+------------------------------------------------------------------------+
+| 9           | RO   (0x0) | **J** (Tentatively reserved for Dynamically Translated Languages       |
+|             |            | extension)                                                             |
++-------------+------------+------------------------------------------------------------------------+
+| 8           | RO   (0x1) | **I** (RV32I/64I/128I base ISA)                                        |
++-------------+------------+------------------------------------------------------------------------+
+| 7           | RO   (0x0) | **H** (Reserved)                                          		    |
++-------------+------------+------------------------------------------------------------------------+
+| 6           | RO   (0x0) | **G** (Additional standard extensions present)                         |
++-------------+------------+------------------------------------------------------------------------+
+| 5           | RO   (0x0) | **F** (Single-precision floating-point extension)                      |
++-------------+------------+------------------------------------------------------------------------+
+| 4           | RO   (0x0) | **E** (RV32E base ISA)                                                 |
```

Comment:
this should depend on the parameter https://github.com/openhwgroup/cve2/blob/main/rtl/cve2_top.sv#L18